pkg/api/types.go

```diff
@@ -116,6 +116,7 @@ type FeatureFlags struct {
 	EnableCSERunInBackground bool `json:"enableCSERunInBackground,omitempty"`
 	BlockOutboundInternet    bool `json:"blockOutboundInternet,omitempty"`
 	EnableIPv6DualStack      bool `json:"enableIPv6DualStack,omitempty"`
+	DisableOutboundSNAT      bool `json:"disableOutboundSnat,omitempty"`
```

Comment:
does this belong in `KubernetesConfig` instead? featureFlags is not meant to be used for k8s configuration but instead allow testing of experimental features without exposing them in docs. The `loadBalancerSku` and `excludeMasterFromStandardLB` knobs are in `KubernetesConfig` for example.

Reply:
If you do move it to `KubernetesConfig`, could you also include it in docs/topics/clusterdefinitions.md?